extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,272 @@ limitations under the License.
+The maximum number of pages to load and show to the user. The maximum number should be less than the total amount of pages. After meeting the number, <amp-next-page> displays the footer recommendation box. The default is `Infinity`.
 
-| Key                   | Value                                                                                                                                                            |
-| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ampUrl` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
-| `title` (required)    | String title of the page, will be used when rendering the recommendation box                                                                                     |
-| `imageUrl` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+### `deep-parsing` (optional)
 
-### Example configuration
+When specified, the `deep-parsing` attribute enables recursive parsing of inline JSON configurations from subsequently loaded documents. This only works if the loaded documents contain an `<amp-next-page>` tag.
 
-The following configuration will recommend two more documents for the user to
-read, and hides the header and footer elements from each child document.
+This behavior is the default if the `<amp-next-page>` component only specifies an inline configuration. If the `<amp-next-page>` component loads the configuration remotely this behavior is disabled by default unless `deep-parsing` is set to `true`.
 
-```json
-{
-  "pages": [
-    {
-      "image": "https://example.com/image1.jpg",
-      "title": "This article shows first",
-      "ampUrl": "https://example.com/article1.amp.html"
-    },
-    {
-      "image": "https://example.com/image2.jpg",
-      "title": "This article shows second",
-      "ampUrl": "https://example.com/article2.amp.html"
-    }
-  ],
-  "hideSelectors": [".header", ".footer"]
-}
-```
+### `xssi-prefix` (optional)
 
-## Substitutions
+When specified, this attribute enables `amp-next-page` to strip a prefix before parsing remotely hosted JSON. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
 
-The `amp-next-page` src allows all standard URL variable substitutions. See
-the [Substitutions Guide](../../spec/amp-var-substitutions.md) for more info.
+## Styling
 
-For example:
+The `<amp-next-page>` component renders a default recommendation box and separator. You may style the appearance of these two elements as follows:
```

Comment:
Please change to:
`The <amp-next-page> component renders a footer recommendation box and separator. You may style these two elements.`